test_webpack/BUILD

```diff
@@ -7,9 +7,12 @@ node_package(
     artifact_paths=["public/bundle"]
 )
 
-docker(
-        name="docker_with_weback_bundle",
-        base_image="python:3.9.1-slim-buster",
-        command=["/bin/bash",],
-        dependencies=[":test_bundle",]
+docker_image(
+    name="docker_with_webpack_bundle",
+    instructions=[
+        "FROM python:3.9.1-slim-buster",
+        "COPY public /public",
```

Comment:
I think this is not exactly the same as the previous source inference based on `node_package.artifact_paths` but it seems like perhaps it gets us where we need to go?